README.md

```diff
@@ -50,23 +50,36 @@ fn main() {
         //.check_ranges(FeatureConfig::Never)                // or look below for an example.
         .build();
```

Comment:
can we add imports to have working example in README?
```rust
use dbc_codegen::Config;
use std::fs::write;
use std::{env::var, path::PathBuf};
```

`dbc_file` should be as a `string` now:
```rs
let dbc_file = std::fs::read_to_string(dbc_path).unwrap();
```